plugins/catalog-backend-module-msgraph/README.md

```diff
@@ -183,3 +183,28 @@ export async function myGroupTransformer(
    }),
  );
 ```
+
+## Dynamically adjust Provider Config
```

Comment:
Could you add a lil bit of text here around what this is and when one might use it?

Reply:
That's a great idea - I will do that on my next push